system/inc/system_dynalib_cloud.h

```diff
@@ -49,6 +49,7 @@ DYNALIB_FN(12, system_cloud, spark_sync_time_pending, bool(void*))
 DYNALIB_FN(13, system_cloud, spark_sync_time_last, system_tick_t(time_t*, void*))
 DYNALIB_FN(14, system_cloud, spark_set_connection_property, int(unsigned, unsigned, particle::protocol::connection_properties_t*, void*))
 DYNALIB_FN(15, system_cloud, spark_set_random_seed_from_cloud_handler, int(void (*handler)(unsigned int), void*))
+DYNALIB_FN(16, system_cloud, spark_send_description, bool(void*))
```

Comment:
Let's paramaterize this with the description type so we can send application, system or vitals (and other types of descriptions.)

Reply:
sounds good!